metricbeat/module/zookeeper/server/server_integration_test.go

```diff
@@ -58,6 +58,15 @@ func TestFetch(t *testing.T) {
 	}
 }
 
+func TestData(t *testing.T) {
+	//t.Skip("Skipping `data.json` generation test")
```

Comment:
I think we don't need this line?

Reply:
Yah, I'm...not sure why that was skipped?